### PR TITLE
RADFISH-191: Update computed form example

### PR DESCRIPTION
**Pull request by gambinish**
Extends https://github.com/NMFS-RADFish/boilerplate/pull/102 PoC with README along with update to example name.

Removes `FormWrapper` and handles form state and submission within `Form`. This moves us away from a "configurable" form implementation.

What should we do about popping the toast message? Previously we were relying on the `FormWrapper` context provider to handle this. Should we create a `ToastContext` at the app level? Or should we include a toast component within the `Form`

https://github.com/NMFS-RADFish/boilerplate/assets/35090461/1e376410-0bac-4f13-a4e7-ebf473f95e7f

### CODE REVIEW

File: examples/computed-form-fields/README.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Computed Form Fields Example
+
+This example includes an example on how to build a form where the values of certain input fields compute the value of a separate readOnly input field elsewhere in the form.
+
+## Steps
```
Comment (brockinit) [inline examples/computed-form-fields/README.md:5]: Is this the README we want? Looks like the on-device storage one.

File: examples/computed-form-fields/src/App.jsx [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  );
+}
+
+const FormInfoAnnotation = () => {
+  return (
+    <Alert type="info" headingLevel={"h1"} heading="Example Form">
+      This is an example of a form with form inputs that compute the values of other form inputs.
+      <br />
+      <br />
+      <strong>Note:</strong> Annotations are for informational purposes only. In production, you
```
Comment (brockinit) [inline examples/computed-form-fields/src/App.jsx:29]: Do we need this note here?

File: examples/computed-form-fields/src/pages/Form.jsx [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+const computedPrice = "computedPrice";
+const speciesData = ["grouper", "salmon", "marlin", "mahimahi"];
+
+const ComputedForm = () => {
+  const [formData, setFormData] = useState({});
+
+  const handleSubmit = (event) => {
+    event.preventDefault();
+    console.log(formData);
+    // TODO: Integrate toast component
```
Comment (brockinit) [inline examples/computed-form-fields/src/pages/Form.jsx:17]: Should this be integrated? Also, need to remove the `console.log`

**Review by brockinit — CHANGES_REQUESTED** [review]
Please take a look at the comments - particularly the README one.

File: examples/computed-form-fields/README.md (hunk 1 above)
Comment (gambinish) [reply examples/computed-form-fields/README.md:5]: Ugh forgot to save and commit. Thanks for catching this

Comment (gambinish) [reply examples/computed-form-fields/README.md:5]: [6ad336f](https://github.com/NMFS-RADFish/boilerplate/pull/104/commits/6ad336fcc8d7584b30b9538809592ffa08e4ac3b)

File: examples/computed-form-fields/src/pages/Form.jsx (hunk 3 above)
Comment (gambinish) [reply examples/computed-form-fields/src/pages/Form.jsx:16]: Removed the console.log. I think the decision from our meeting yesterday was that we still were undecided about how we should implement the `Toast`. eg Should it be implemented in a `ToastProvider` through `packages` or should it be implemented within each component.

Either way, I think we should probably just implement the common pattern throughout all the examples holistically if that's alright with you

File: examples/computed-form-fields/src/App.jsx (hunk 2 above)
Comment (gambinish) [reply examples/computed-form-fields/src/App.jsx:29]: I can remove the bit about "information purposes only" good call

Comment (gambinish) [reply examples/computed-form-fields/src/App.jsx]: [7b12458](https://github.com/NMFS-RADFish/boilerplate/pull/104/commits/7b124583791812b37bd9d56d42b80d80ede055a8)

File: examples/computed-form-fields/src/pages/Form.jsx (hunk 3 above)
Comment (brockinit) [reply examples/computed-form-fields/src/pages/Form.jsx:16]: You're right. And yes, let's implement the common pattern though all examples.

**Review by brockinit — APPROVED** [review]

**Pull request merged**